src/shipit_code_coverage/tests/conftest.py

```diff
@@ -120,3 +131,21 @@ def grcov_uncovered_function_artifact():
 def jsvm_uncovered_function_artifact():
     with generate_coverage_artifact('jsvm_uncovered_function.info') as f:
         yield f
+
+
+@pytest.fixture(scope='session')
+def mock_get_commit(MERCURIAL_COMMIT):
```

Comment:
No need to have separate fixtures, add the responses.add call directly in the MERCURIAL_COMMIT fixture

Reply:
Done